### DISCUSSION TITLE
Version 3.0.0 is out!

### DISCUSSION THREAD

**Discussion by estruyf** [estruyf/vscode-front-matter#71] [Announcements]
I just released version `3.0.0` of Front Matter with the following new features:

- [#61](https://github.com/estruyf/vscode-front-matter/issues/61): List of recently modified files
- [#64](https://github.com/estruyf/vscode-front-matter/issues/64): Publish toggle for easier publishing an article
- [#65](https://github.com/estruyf/vscode-front-matter/issues/65): Aggregate articles in draft
- [#66](https://github.com/estruyf/vscode-front-matter/issues/66): New dashboard webview on which you can manage all your content
- [#69](https://github.com/estruyf/vscode-front-matter/issues/69): Welcome screen for getting started

Get it at: https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-front-matter

## Welcome screen

![Screenshot 2021-08-27 at 11 02 19](https://user-images.githubusercontent.com/2900833/131114826-924c5bfc-f209-47e6-801c-84a6869eed58.png)

## Dashboard

![Screenshot 2021-08-26 at 11 21 05](https://user-images.githubusercontent.com/2900833/131114847-17bda242-7f77-4f89-b4eb-8dc3e7c2ca47.png)

## List of recently modified files

![Screenshot 2021-08-26 at 11 32 38](https://user-images.githubusercontent.com/2900833/131114880-5186f5a2-0ce5-4d91-b7d8-0fff445f33e6.png)

## Publish toggle

![Screenshot 2021-08-26 at 11 39 08](https://user-images.githubusercontent.com/2900833/131114918-2db3e92b-19be-40ba-af42-9d32f3e05827.png)

**I hope you like this new release!**

*Feel free to consider [sponsoring](https://github.com/sponsors/estruyf) this project and/or [writing a review](https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-front-matter&ssr=false#review-details)*

**Comment by Fasani** [comment]
The addition of `draft: true` is nice, before I managed a `_temp` folder but having the draft in the frontmatter is quite elegant and my build pipeline in Gatsby can easily be configured to ignore draft articles.
